styles.css

```diff
@@ -183,14 +220,15 @@ li a:hover {
 */
 
 .vision-mission-values {
+  /* border: 1px red solid; */
   display: grid;
   gap: 2rem;
   grid-template-columns: 1fr;
```

Comment:
The designs requires that you display the cards in two rows and two cols on desktop. Not one. Refigure this out.